docs/docs/user/references/cli-commands.md

```diff
@@ -70,6 +71,7 @@ $ kpops deploy [OPTIONS] PIPELINE_PATH [COMPONENTS_MODULE]
 * `--dry-run / --execute`: Whether to dry run the command or execute it  [default: dry-run]
 * `--verbose / --no-verbose`: Enable verbose printing  [default: no-verbose]
 * `--steps TEXT`: Comma separated list of steps to apply the command on  [env var: KPOPS_PIPELINE_STEPS]
+* `--filter-type [include|exclude]`: If the --steps option should include/exclude the steps  [default: include]
```

Comment:
order should be consistent with other operations: `--steps`, `--filter-type`, `--dry-run`, `--verbose`